rs/rust_canisters/canister_test/src/canister.rs

```diff
@@ -920,6 +920,10 @@ pub struct Install<'a> {
     pub mode: CanisterInstallMode,
     pub runtime: &'a Runtime,
     pub wasm: Wasm,
+    // The compute and memory allocation fields are still useful here as they
+    // are used to set the allocation of the canister when an `update_settings`
+    // request is used. However, it's weird that they are in a struct called
+    // `Install`. This should probably be cleaned up in the future.
```

Comment:
Can you say that these are not used when installing, but are used to do `update_settings`? "are still useful" probably does not make too much sense to someone outside this PR.

Reply:
Rephrased to be more explicit about how they're used and why that's weird and should be cleaned up.